apps/web/src/pages/integrations/components/ChannelGroup.tsx

```diff
@@ -18,15 +18,51 @@ export function ChannelGroup({
   }
 
   return (
-    <Grid mb={50}>
-      <Grid.Col span={12} data-test-id={`integration-group-${title.toLowerCase()}`}>
+    <Wrapper>
+      <div data-test-id={`integration-group-${title.toLowerCase()}`}>
         <Title size={2}>{title}</Title>
-      </Grid.Col>
-      {providers.map((provider) => (
-        <Grid.Col sm={6} md={3} xs={4} key={provider.providerId}>
-          <ProviderCard provider={provider} onConnectClick={handlerOnConnectClick} />
-        </Grid.Col>
-      ))}
-    </Grid>
+      </div>
+      <ContentWrapper>
+        {providers.map((provider) => (
+          <div key={provider.providerId}>
+            <ProviderCard provider={provider} onConnectClick={handlerOnConnectClick} />
+          </div>
+        ))}
+      </ContentWrapper>
+    </Wrapper>
   );
 }
+
+const LAYOUT_GAP = 25;
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${LAYOUT_GAP}px;
+  margin-bottom: 50px;
+`;
+
+const ContentWrapper = styled.div`
```

Comment:
```suggestion
const ContentWrapper = styled(Container)`
```
Here the same, as I would expect the Mantine component to pass all the grid CSS properties to the `div` element. But not 100% sure.